### PR TITLE
Kommander 1.1 Beta Docs

### PR DESCRIPTION
**Pull request by juliangieseke**
ℹ️ reviewing per commit is probably easier than trying to review all at once

## Jira Ticket

https://jira.d2iq.com/browse/D2IQ-67840

## Description of changes being made
- renamed "latest" to 1.0 and added new 1.1 beta docs
- added basic changelogs for kommander 1.1. betas

## Checklist
- [x] Make sure you change all affected versions (e.g. 1.10, 1.11, 1.12, 1.13).
- [ ? ] Test all commands and procedures where applicable.
- [x] Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects) if you are moving a page.

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing) for more information.

### CODE REVIEW

**Comment by juliangieseke** [comment]
I've tried adding 1.1 beta docs (esp. release notes), does this look correct?

**Comment by cmastr** [comment]
@juliangieseke path must be 1.1.0-beta to build properly. Won't build locally for me.

**Comment by mperrotti** [comment]
@juliangieseke - should I just add the quota docs to this PR instead of creating a new one?

**Comment by juliangieseke** [comment]
@mperrotti Please create a new one.

**Comment by juliangieseke** [comment]
@cmastr keeping this PR up to date while work on docs continue will get tedious pretty fast, what is necessary to get it merged?

File: pages/ksphere/kommander/1.1.0-beta/clusters/attach-cluster/index.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  tls.crt: $(base64 -w0 < ${CA_CERT_FILE})
+  tls.key: $(base64 -w0 < ${PRIV_KEY_FILE})
+EOF
+```
+
+After the CA secret has been created successfully, a custom kubeconfig can be retrieved by visiting the `/token` endpoint on the Kommander cluster domain. Selecting the attached cluster name displays the instructions to assemble a kubeconfig for accessing its Kubernetes API.
+
+## Attaching Kubernetes Cluster Using a New Service Account
+
+When connecting an existing cluster to Kommander a separate special service account should be created to keep that access specific and isolated to Kommander.
```
Comment (russurquhart1) [inline pages/ksphere/kommander/1.1.0-beta/clusters/attach-cluster/index.md:56]:
```suggestion
When connecting an existing cluster to Kommander a separate special service account must be created to keep that access specific and isolated to Kommander.
```

File: pages/ksphere/kommander/1.1.0-beta/clusters/attach-cluster/index.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+To get started, ensure you have [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) set up and configured with [ClusterAdmin](https://kubernetes.io/docs/concepts/cluster-administration/cluster-administration-overview/) for the cluster you intend to connect to Konvoy.
```
Comment (russurquhart1) [inline pages/ksphere/kommander/1.1.0-beta/clusters/attach-cluster/index.md:58]:
```suggestion
To get started, ensure you have [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) set up and configured with [ClusterAdmin](https://kubernetes.io/docs/concepts/cluster-administration/cluster-administration-overview/) for the cluster you want to connect to Konvoy.
```

**Comment by juliangieseke** [comment]
@russurquhart1 given I have not changed any text, your suggestions would nee to be fixed/changed in 1.0 and 1.1 docs, can we do them in a separate PR after this one got merged?

File: docker/nginx/redirects-307.map [hunk 3]
```diff
@@ -1,6 +1,10 @@
 ~^/mesosphere/dcos/latest/(.*)$ /mesosphere/dcos/2.0/$1;
 ~^/ksphere/konvoy/latest/(.*)$ /ksphere/konvoy/1.4/$1;
+<<<<<<< HEAD
```
Comment (mperrotti) [inline docker/nginx/redirects-307.map:3]: @juliangieseke - I think some merge conflicts were  accidentally committed

**Comment by juliangieseke** [comment]
rebased, fixed and updated a couple more details. `npm run dev` successfully runs on my machine.

**Comment by juliangieseke** [comment]
should be ready to merge now @cmastr 👍

**Comment by juliangieseke** [comment]
rebased

**Comment by cmastr** [comment]
@jackson ran locally and this is passing now. merging!

**Pull request merged**